change/@fluentui-react-file-type-icons-d3198708-7205-47f9-8ef1-3eff3ae33af5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26990&commitId=641e5d3bcce0b69238130daca2f75ddb7e179dc4&env=prod)

✅ There was 9 screenshots added, 0 screenshots removed, 1038 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|0| Added|
|FocusTrapZones.Panel on its own.default.chromium.png|0| Added|
|FolderCover.Large Default Cover.default.chromium.png|0| Added|
|FolderCover.Large Media Cover.default.chromium.png|0| Added|
|FolderCover.Small Default Cover.default.chromium.png|0| Added|
|Panel.Large right.default.chromium.png|0| Added|
|Panel.Medium right - RTL.default.chromium.png|0| Added|
|Panel.Medium right.default.chromium.png|0| Added|
|Panel.Small fluid right.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

Reply:
@bigbadcapers This might be caused by your branch being outdated, I'd merge with latest master and see the results from there.

Reply:
ok, ty, the checks are re-running. ill resolve this one and see if it shows up again.